src/components/browse/components/FacetList/FacetTermsList.js

```diff
@@ -370,14 +420,34 @@ const ListOfTerms = React.memo(function ListOfTerms(props){
             );
         }
 
+        // show simple text input for basic search (search within returned values)
+        // or show SAYT control if item search is available
+        let facetSearch = null;
+        if (searchType === 'basic') {
+            facetSearch = (
+                <div style={{ 'padding': '10px', 'fontSize': '0.875rem' }}>
+                    <input className="form-control" autoComplete="off" type="search" placeholder="Search"
+                        name="q" onChange={onBasicTermSearch} key="facet-search-input" />
+                </div>);
+        } else if ((searchType === 'sayt') || (searchType === 'sayt_without_terms')) {
+            const itemType = facet.sayt_item_type && typeof facet.sayt_item_type === 'string' && facet.sayt_item_type !== '' ? facet.sayt_item_type : 'Item';
+            const baseHref = "/search/?type=" + itemType;
+            facetSearch = (
+                <div className="d-flex flex-wrap" style={{ 'padding': '10px', 'fontSize': '0.875rem' }}>
```

Comment:
Use className "p-2" (or similar, depending on padding desired / what looks good) and className "small" or "text-small" or "text-smaller" instead of defining styles inline.

Inline styles, especially in JSX/React -- [where performance is negatively impacted by it generally](https://reactjs.org/docs/faq-styling.html#are-inline-styles-bad) -- but also in general/html, are discouraged whenever they can be avoided.